app/views/content/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs.md

```diff
@@ -45,4 +45,4 @@ You need [specific qualifications to teach a class of pupils with hearing impair
 
 A special educational needs coordinator (SENCO) assesses, plans and monitors the progress of disabled pupils and pupils with special educational needs.
 
-Once you’re a qualified teacher you'll need to complete the [National Award in Special Educational Needs Coordination (NASENCo)](https://nasen.org.uk/page/nasenco) when you take up your SENCO post.
+Once you’re a qualified teacher you'll need to complete further training when you take up your post. [Visit the National Award in Special Educational Needs Coordination website for more on SENCO training](https://nasen.org.uk/page/nasenco).
```

Comment:
``Once you’re a qualified teacher, you'll need to complete further training when you take up your post. [Visit the National Award in Special Educational Needs Coordination website for more on SENCO training](https://nasen.org.uk/page/nasenco).